File: src/Commands/Music/PlayCommand.js
const { Command } = require('../../')

module.exports = class PlayCommand extends Command {
    constructor(client) {
        super(client, {
            name: 'play',
            aliases: ['p'],
            description: 'Toca a música de sua escolha no canal de voz',
            usage: '<prefix>play <args>',
            category: 'Music',
            cooldown: 8
        })
    }

    execute(client, msg, zoe, args, t) {
    
    }
}

File: src/main/kotlin/com/github/mrdroox/projects/commands/music/PlayCommand.kt
package com.github.mrdroox.projects.commands.music

import com.github.mrdroox.projects.utilities.Command
import com.github.mrdroox.projects.utilities.CommandCategory
import com.github.mrdroox.projects.utilities.CommandContext

class PlayCommand: Command() {
    override var name: String? = "play"
    override var aliases = listOf("p", "tocar")
    override var category = CommandCategory.MUSIC

    override fun execute(ctx: CommandContext) {

    }
}